test/which.js

```diff
@@ -23,8 +23,16 @@ assert.equal(result, null);
 // Valids
 //
 
-var result = shell.which('node');
+var node = shell.which('node');
 assert.equal(shell.error(), null);
-assert.equal(fs.existsSync(result), true);
+assert.equal(fs.existsSync(node), true);
+
+if (process.platform === 'win32') {
+    // This should be equivalent on Windows
+    var nodeExe = shell.which('node.exe');
+    assert.equal(shell.error(), null);
+    assert.equal(fs.existsSync(nodeExe), true);
```

Comment:
I removed this because it's redundant after the initial check (if this fails after the last, it's not a bug in this).